webrtc.html

```diff
@@ -2559,6 +2560,25 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpTransceiver createRtpTransceiver((MediaStreamTrack or
+          DOMString track_or_kind), RTCRtpTransceiverInit init)</dt>
```

Comment:
s/track_or_kind/trackOrKind/ ? We don't use underscore as a separator in any other identifiers.